keyboards/underscore33/keymaps/default_r1/keymap.c

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2020 tominabox1
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum layers{
+  _BASE,
+  _NUM_SYM,
+  _NAV
+};
+
+enum custom_keycodes{
+  RGBRST = SAFE_RANGE,
+};
+
+#define KC_NUM_SPC LT(_NUM_SYM, KC_SPC)
+#define KC_NAV_ENT LT(_NAV, KC_ENT)
+#define KC_GA LGUI_T(KC_A)
+#define KC_AS LALT_T(KC_S)
+#define KC_CD LCTL_T(KC_D)
+#define KC_SF LSFT_T(KC_F)
+#define KC_SJ RSFT_T(KC_J)
+#define KC_CK RCTL_T(KC_K)
+#define KC_AL RALT_T(KC_L)
+#define KC_GSCLN RGUI_T(KC_SCLN)
+
+enum combo_events {
+  COMBO_BSPC,
+  COMBO_NUMBAK,
+  COMBO_TAB,
+  COMBO_ESC,
+  COMBO_DEL,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT_33_split_space(
```

Comment:
This is reporting that it is missing a key.
```
keyboards/underscore33/keymaps/default_r1/keymap.c:53:3: error: macro "LAYOUT_33_split_space" requires 36 arguments, but only 35 given
```

Reply:
Travis reports it but if you build the keymap it works, I don't know what else to do to fix that besides have totally different layout macros for each revision, which I was trying to avoid for people with existing keymaps

Reply:
Unfortunately, I think that may be best. 

The biggest concern that I have is the cases where it will fail.  Especially for somebody that isn't familiar with coding, as well as somebody using the QMK Configurator.